src/main/java/uk/ac/ebi/eva/contigalias/controller/AdminController.java

```diff
@@ -47,10 +48,18 @@ public AdminController(AssemblyService service) {
         this.service = service;
     }
 
-    @ApiOperation(value = "Get or fetch an assembly using its Genbank or Refseq accession.")
+    @ApiOperation(value = "Get or fetch an assembly using its Genbank or Refseq accession.",
+            notes = "Given an assembly's accession this endpoint will return an assembly that matches that accession." +
+                    " The accession can be either a genbank or refseq accession and the software will automatically " +
+                    "fetch a result from the database for any assembly having the accession as it's genbank or refseq" +
+                    " accession. This endpoint will first look for the assembly in local database and return the " +
+                    "result. If local search fails, it search for the target assembly at a remote source (NCBI by " +
+                    "default). If desired assembly is found at remote source, it will fetch and add it to the local " +
+                    "database and also return the result to you." +
+                    "This endpoint will either return a single result or an HTTP Response with error code 404.")
     @GetMapping(value = "v1/assemblies/{accession}", produces = "application/json")
     public ResponseEntity<List<AssemblyEntity>> getAssemblyOrFetchByAccession(
-            @PathVariable String accession, @RequestParam(required = false) Integer page,
+            @PathVariable @ApiParam(value = "Genbank or Refseq assembly accession. Eg: GCA_000001405.10") String accession, @RequestParam(required = false) Integer page,
             @RequestParam(required = false) Integer size) throws IOException {
```

Comment:
can you add descriptions for page and size too? just short sentences that say it's the page size, not a hypothetical assembly size.